multimodal/src/autogluon/multimodal/utils.py

```diff
@@ -27,6 +29,11 @@
 from sklearn.preprocessing import LabelEncoder
 from torch import nn
 
+try:
+    import xml.etree.cElementTree as ET
```

Comment:
Do we need to install the package xml?

Reply:
xml.etree has a security problem, have changed it to defusedxml and added in setup.py